sdv/constraints/tabular.py

```diff
@@ -294,13 +308,32 @@ def reverse_transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        diff = (np.exp(table_data[self._high]).round() - 1).clip(0)
-        low_column = table_data[self._low]
+        diff = (np.exp(table_data[self._diff_column]).round() - 1).clip(0)
 
-        if pd.api.types.is_datetime64_ns_dtype(low_column):
-            diff = pd.to_timedelta(diff)
+        if self._drop == 'high':
+            low_column = table_data[self._low]
+            if pd.api.types.is_datetime64_ns_dtype(low_column):
+                diff = pd.to_timedelta(diff)
+
+            table_data[self._high] = (low_column + diff).astype(self._dtype)
+
+        elif self._drop == 'low':
+            high_column = table_data[self._high]
+            if pd.api.types.is_datetime64_ns_dtype(high_column):
+                diff = pd.to_timedelta(diff)
+
+            table_data[self._low] = (high_column - diff).astype(self._dtype)
+
+        else:
+            low_column = table_data[self._low]
+            if pd.api.types.is_datetime64_ns_dtype(low_column):
+                diff = pd.to_timedelta(diff)
+
+            invalid = ~self.is_valid(table_data)
+            new_high_values = low_column.loc[invalid] + diff.loc[invalid]
+            table_data[self._high].loc[invalid] = new_high_values.astype(self._dtype)
 
-        table_data[self._high] = (low_column + diff).astype(self._dtype)
+        table_data = table_data.drop(self._diff_column, axis=1)
```

Comment:
a `del table_data[self._diff_column]` would also do the trick, but I do not know which one is faster and has less memory overhead. @amontanez24 could we time this and also research a bit whether one is preferable over the other one?
@pvk-developer any thoughts?

Reply:
looks like `drop` is faster https://stackoverflow.com/questions/47426089/python-del-vs-pandas-drop/51194789